### PR TITLE
Add FB functionality to UserHome, Created new FB function

### LINKED ISSUE #59: Add functionality to the available reservations page

### LINKED ISSUE #64: Change available reservations to live on one line

### PR DESCRIPTION
**Pull request by plstarkey18**
For this pull request, I added FB functionality to the UserHome page, which is now in a datatable format compared to the expansion panels. I decided to create a new function that requested deviceName, deviceTag, and min and max duration that can be displayed for the user. I have a plan that once the user clicks the reserve button, then it will grab the deviceTag and query the availability in the review reservation page. I created a function to make sure that once the button is clicked, it will log in the console the right item. I also made minor changes to the ReviewRes component, but it is only design and not functionality.

### CODE REVIEW

File: src/components/User/UserHome.vue [hunk 1]
```diff
@@ -1,75 +1,94 @@
...
+        const inventory = await getAvailableReservations();
+        this.devices = inventory;
+      } catch (e) {
+        console.log(e);
+      }
+    },
+    selectItem(item) {
+      this.selctedIndex = this.devices.indexOf(item);
+      this.selectedItem = { ...item };
+      console.log(this.selectedItem.deviceName);
```
Comment (greyhixson) [inline src/components/User/UserHome.vue]:
```suggestion
      
```

**Review by greyhixson — CHANGES_REQUESTED** [review]
Looks good except for the console.log. Similarly to everyone elses right now we have a bunch of v-data-tables that all look the same. Next sprint everyone will need to style them, so they are dissimilar and friendlier to use.

**Comment by plstarkey18** [comment]
After I fixed your requested changes, I updated my branch to reflect Brian's merge with master. Should be up to date now.

**Review by greyhixson — APPROVED** [review]
Great job!

**Pull request merged**